### ISSUE TITLE
unable to find image 'obolibrary/odkfull:latest' locally

### ISSUE THREAD

**Issue by srobb1** [INCATools/ontology-development-kit#127]
I think I must be doing something wrong. I have downloaded a new copy of ODK. I tried both the master branch from today and the most current release (1.1.4).

The docs say to start with ./seed-via-docker.sh

I try and I get this error even when I just run it with '-h'

```
./seed-via-docker.sh -h
Unable to find image 'obolibrary/odkfull:latest' locally
docker: Error response from daemon: Get https://registry-1.docker.io/v2/obolibrary/odkfull/manifests/latest: unauthorized: incorrect username or password.
See 'docker run --help'.
```

Am I missing a step? I have set this up before, I am pretty sure I followed the instructions in the README.md when I did this last.  

Thank you,
Sofia

**Comment by srobb1** [comment]
```
docker run hello-world
Unable to find image 'hello-world:latest' locally
docker: Error response from daemon: Get https://registry-1.docker.io/v2/library/hello-world/manifests/latest: unauthorized: incorrect username or password.
See 'docker run --help'.
```

**Comment by cmungall** [comment]
Seems like a not uncommon occurrence, do any of these help?

https://www.google.com/search?q=docker%3A+Error+response+from+daemon%3A+Get+https%3A%2F%2Fregistry-1.docker.io%2F&oq=docker%3A+Error+response+from+daemon%3A+Get+https%3A%2F%2Fregistry-1.docker.io%2F

**Comment by srobb1** [comment]
I tried restarting docker, logging out, logging back in no luck. I double checked that my login was corrent. I could successfully log in online. When I ran this on the command line 

```
docker login
``` 

and logged in it worked.

**Comment by matentzn** [comment]
wow, good to know. I am sure now that more and more people are using this, these kinds of issues will keep popping up. So is everything okay now?

**Comment by srobb1** [comment]
Yup downloading everything now. @cmungall Thank you for sending that link. 

One last thing I noticed. I can log into docker.com with my username (srobb1) and my email address. After playing around with this I found that I can only log in on the command line with my username and not my email address. While the docker app allows me to log in with both. I think this was my issue. I am not 100% sure what I was logged in with in the app on my computer earlier, but I suspect it was with my email address.
